### ISSUE TITLE
We tried to use `record: true` within our circle ci yaml file, we're on a free tier as we're an open source project and get the following error when we run our build:

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/circleci-orb#50]
```
#!/bin/bash -eo pipefail
npx cypress run \
   \
   \
   --record \
    --group 'default group' \
     \
Grouping is not included under your current billing plan.

To run your tests with groups, please visit your billing and upgrade to another plan with grouping.
...
```

We don't have a group set, is it possible to do this on the free tier?

Links to our project ci server and github repository are below:

https://circleci.com/gh/build-canaries/nevergreen/1224

https://github.com/build-canaries/nevergreen/blob/51e67aca6bb52d2f89cf3e2da820fa6d074476b4/.circleci/config.yml#L184

_Originally posted by @cowley05 in https://github.com/cypress-io/circleci-orb/issues/43#issuecomment-443460454_

**Comment by bahmutov** [comment]
we should not set group name when recording, because this assumes paid plan with parallelization support

**Comment by bahmutov** [comment]
published as `cypress-io/cypress@1.1.2`
